django/website/main/static/scss/logframe.scss

```diff
@@ -842,52 +857,57 @@ div.activity-item {
     width: 85%;
 }
 
-.edit-link a, .logframe-link a {
+span.organization-name {
+	width: 85%;
+	display: inline-block;
+}
+
+.edit-link a, .logframe-link a, .organization-name {
     padding: 1.5em;
     width: 100%;
     display: block;
 }
 
-.logframe-edit h2 {
+.logframe-edit h2, .organization-edit h2 {
     margin-bottom: 1.5em;
     margin-left: 10%;
     margin-top: 0;
 }
 
-.logframe-edit .pure-form-stacked .help-text {
+.logframe-edit .pure-form-stacked .help-text, .organization-edit .pure-form-stacked .help-text {
```

Comment:
There's quite a lot of duplication here, which seems like it might be worth either a common additional class for those elements that already have class 'logframe-edit' or 'organization-edit', or a better sass structure such as 

```
.logframe-edit,
organization-edit {
    .pure-form-stacked {
        .help-text {
            ...
        }
    }
}
```

Also use child (>) rather than descendant selectors if possible